src/test-run/commands/actions.js

```diff
@@ -303,17 +303,17 @@ export class NavigateToCommand extends Assignable {
     constructor (obj) {
         super(obj);
 
-        this.type = TYPE.navigateTo;
-        this.url  = null;
-        this.storages = null;
+        this.type          = TYPE.navigateTo;
+        this.url           = null;
+        this.stateSnapshot = null;
 
         this._assignFrom(obj, true);
     }
 
     _getAssignableProperties () {
         return [
             { name: 'url', type: urlArgument, required: true },
-            { name: 'storages', type: nullableStringArgument }
+            { name: 'stateSnapshot', type: nullableStringArgument }
```

Comment:
`stateSnapshot` isn't 'string or null' anymore, is it?

Reply:
still is

Reply:
[yep](https://github.com/DevExpress/testcafe/pull/2285/files#diff-ef5c498529d63a584dbbfbbeeada71e1R110), I see now